### DISCUSSION TITLE
Contour Shuttle Compatibility with SE

### DISCUSSION THREAD

**Discussion by Hayholten** [SubtitleEdit/subtitleedit#5919] [Q&A]
Hello to all;

I'm opening this topic to see if any of you were using a ShuttlePro v2 (or Shuttle Xpress) controller with Subtitle Edit.

[https://www.contourdesign.ca/product/shuttlepro-v2/](https://www.contourdesign.ca/product-category/workflow/)

I'm planning to get one and I wanted to know if SE is perfectly compatible with the controller.

Thanks for your feedback.

Thibault

**Comment by Hayholten** [comment]
Hello again!

No one?

**Comment by ghalymt** [comment]
I do not know if this will help you, but I use subtitle edit with this mouse.
I create my own macros and assign it to the 16 side keys. With time, working on subtitles with this mouse becomes very fast and accurate.

https://www.amazon.com/Redragon-Wireless-Programmable-Buttons-Battery/dp/B088B3ZM76/ref=sr_1_3?crid=V5YRTTMI5TE7&keywords=m913&qid=1651153194&sprefix=m913%2Caps%2C173&sr=8-3

**Reply by Hayholten** [reply]
Hello @ghalymt !

Thank you very much for your answer.

Indeed, I already have a programmable mouse with several configuration options. It's very useful but what interests me most is being able to navigate in the timeline with the ShuttlePro v2 jog.

I have already used it on another subtitling software and it is a revolution in terms of time saving^^

My hunch is that it will be compatible but I'd rather be 100% sure before I make the purchase ;)
